deeprankcore/dataset.py

```diff
@@ -398,12 +539,12 @@ def __init__( # pylint: disable=too-many-arguments, too-many-locals
         classes: Optional[Union[List[str], List[int], List[float]]] = None,
         tqdm: Optional[bool] = True,
         root: Optional[str] = "./",
-        transform: Optional[Callable] = None,
-        pre_transform: Optional[Callable] = None,
-        edge_features_transform: Optional[Callable] = lambda x: np.tanh(-x / 2 + 2) + 1,
+        standardize: bool = False,
         target_transform: Optional[bool] = False,
         target_filter: Optional[Dict[str, str]] = None,
-        check_integrity: bool = True
+        check_integrity: bool = True,
+        train: bool = True,
+        dataset_train: GraphDataset = None
```

Comment:
Why are these only part of `GraphDataset` and not `GridDataset` (or parent class)?

Reply:
For the GridDataset more work is needed, and I opened two issues about it (#351)

Reply:
> For the GridDataset more work is needed, and I opened an issue about it (#351)